docs/01-tooling/03-fast-and-isolated-JS-unit-tests.md

```diff
@@ -2,10 +2,159 @@
 
 ### Intro
```

Comment:
This is an intro but the topic seems to be more about QUnit tests in the MediaWiki environment than an introduction to the article. Consider renaming this section to something like "old way" or "Conventional QUnit MediaWiki tests."

Reply:
I've added a second paragraph sumarizing and the problems are explained so I think that works.

We could consider adding a TLDR to all the articles at the top. What do you think?

Reply:
Sounds good, will do as a followup!